examples/sample_client.py

```diff
@@ -19,9 +24,15 @@ def _usage():
 
 client = Uid2Client(base_url, auth_key, secret_key)
 keys = client.refresh_keys()
-result = decrypt(ad_token, keys)
-
-print('UID2 =', result.uid2)
-print('Established =', result.established)
-print('Site ID =', result.site_id)
-print('Site Key Site ID =', result.site_key_site_id)
+decrypt_result = decrypt(ad_token, keys)
+
+print('UID2 =', decrypt_result.uid2)
+print('Established =', decrypt_result.established)
+print('Site ID =', decrypt_result.site_id)
+print('Site Key Site ID =', decrypt_result.site_key_site_id)
+
+# Not required for DSPs but if you are using UID2 Sharing functionality then this is how to encrypt raw UID2 into
```

Comment:
# Not required for DSPs, but for those using UID2 sharing functionality this shows how to encrypt a raw UID2 into

Reply:
fixed